Header header

string[] sensor_ids
int32[] images_acquired
